bodhi/README.md

```diff
@@ -15,17 +15,17 @@ npm install @acala-network/bodhi
 ```ts
 import { WsProvider } from "@polkadot/api";
 import { createTestPairs } from "@polkadot/keyring/testingPairs";
-import { PolkaSigner, SignerProvider, Signer } from "@acala-network/bodhi";
+import { SubstrateSigner, SignerProvider, Signer } from "@acala-network/bodhi";
 
 const provider = new SignerProvider({
   provider: new WsProvider("ws://localhost:9944")
 });
 
-// create a testing polkaSigner, in prod this is usually an extension signer
+// create a substrate payload signer, dapps will use an extension signer
 const { alice } = createTestPairs();
-const polkaSigner = new PolkaSigner(provider.api.registry, [alice]);
+const signer = new SubstrateSigner(provider.api.registry, alice);
 
-const wallet = new Signer(provider, alice.address, polkaSigner);
+const wallet = new Signer(provider, alice.address, signer);

```

Comment:
`new Signer(..., signer)` is a bit confusing, maybe `new Signer(..., substrateSigner)`?

Reply:
not a big deal... either way